storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1374,5 +1393,29 @@ public static RuntimeException wrapInRuntime(Exception e){
             return new RuntimeException(e);
         }
     }
+
+    public static int getAvailablePort(Integer port) {
+        Integer localPort = null;
+        ServerSocket socket = null;
+        try {
+            socket = new ServerSocket(port);
+            localPort = socket.getLocalPort();
+        } catch (IOException exp) {
+            getAvailablePort(0);
+        } finally {
```

Comment:
If it is an int not an Integer we don't have to wrap all of the calls to it with `(Integer. )`